doc/source/user/c-info.python-as-glue.rst

```diff
@@ -906,7 +906,7 @@ The ``code.c`` file also contains the function ``dfilter2d``:
        a has strides that are multiples of sizeof(double)
     */
     void
-    dfilter2d(double *a, double *b, int *astrides, int *dims)
+    dfilter2d(double *a, double *b, int *astrides, ssize_t *dims)
```

Comment:
`astrides` should also be of `ssize_t` type. And there is not much benefit in modifying the type of the function parameter if inside the function you still use `M` and `N` (and `S0` and `S1` once `astrides` is also changed) as `int`s.